cmake/modules/Utility.cmake

```diff
@@ -88,6 +88,17 @@ macro(add_llbuild_library name)
 
   set_output_directory(${name} ${LLBUILD_EXECUTABLE_OUTPUT_INTDIR} ${LLBUILD_LIBRARY_OUTPUT_INTDIR})
 
+  if (MSVC)
+    if(ARG_OUTPUT_NAME)
+      set_target_properties(${name}
+        PROPERTIES
+        PDB_NAME lib${ARG_OUTPUT_NAME})
+	else()
+      set_target_properties(${name}
+        PROPERTIES
+        PDB_NAME lib${name})
+    endif()
+  endif()
```

Comment:
Now on Windows, this produces:
bin/llbuild.exe
bn/llbuild.pdb
bin/libllbuild.pdb
lib/llbuild.lib
lib/llbuildSwift.lib

And -lllbuild or -lllbuildswift will work across platforms. Darwin/Linux should be unchanged.

Reply:
Don't think that you need the `MSVC` check.  Plus, clang is able to generate PDBs as well.  Can you just default the parameter instead to avoid the conditional?

```
if(NOT ARG_OUTPUT_NAME)
  set(ARG_OUTPUT_NAME ${name})
endif()
set_target_properties(${name} PROPERTIES PDB_NAME lib${ARG_OUTPUT_NAME})
```